karma.conf.js

```diff
@@ -1,7 +1,7 @@
 // Karma configuration file, see link for more information
 // https://karma-runner.github.io/1.0/config/configuration-file.html
 
```

Comment:
This file was supposed to be left untouched.  You will need to set it back via a new commit before I can accept this in.

Reply:
committed the changes